### ISSUE TITLE
CsWinRT crashes for AsAsyncOperation marshalling

### ISSUE THREAD

**Issue by huoyaoyuan** [microsoft/CsWinRT#1165]
**Describe the bug**
https://github.com/microsoft/microsoft-ui-xaml/issues/6883

**To Reproduce**
When returning from `ISupportIncrementalLoading.LoadMoreItemsAsync`, it throws different exceptions, based on the call stack.

I'm not 100% sure for the root cause, but since it's fixed by pinning CsWinRT, the marshalling should be most suspective.

**Version Info**
It works fine when pinning `Microsoft.Windows.SDK.NET.Ref` to `10.0.19041.22` (contains WinRT.Runtime 1.4), and fails when using `10.0.19041.23` (contains WinRT.Runtime 1.5).

**Additional context**

**Comment by manodasanW** [comment]
I was able to repro this issue.  This looks to be a duplicate of #1124 and the fix for this is in the upcoming .NET SDK update.  In the meantime, you should be able to unblock yourself by temporarily adding an explicit reference to the **.24** version of Windows SDK projection.  You can do that by adding the following property if you target 19041:

`<WindowsSdkPackageVersion>10.0.19041.24</WindowsSdkPackageVersion>`

**Comment by huoyaoyuan** [comment]
Confirmed fixed with SDK 6.0.202.
